src/routes/(app)/+layout.svelte

```diff
@@ -113,14 +114,49 @@
 
 	$: logoClass = data.logo.isWide ? 'h-[60px] w-auto' : 'h-[60px] w-[60px] rounded-full';
 	const { t, locale, textAddress } = useI18n();
+	let ageWarning: false;
 </script>
 
 <!--
 	We use data-sveltekit-preload-data="off" on header/footer links due to this: 	https://github.com/sveltejs/kit/issues/9508#issuecomment-1807200239
 -->
 
 <div data-sveltekit-preload-data={data.isMaintenance ? 'tap' : 'hover'} style="display: contents;">
-	{#if $page.data.layoutReset || $page.url.searchParams.get('display') === 'headless'}
+	{#if data.age.restriction && !data.sessionAcceptAgeLimitation}
+		<form class="mx-auto max-w-7xl" method="POST" action="?/navigate">
+			{#if data.cmsAgewall && data.cmsAgewallData}
+				<CmsDesign
+					challenges={data.cmsAgewallData.challenges}
+					tokens={data.cmsAgewallData.tokens}
+					sliders={data.cmsAgewallData.sliders}
+					products={data.cmsAgewallData.products}
+					pictures={data.cmsAgewallData.pictures}
+					tags={data.cmsAgewallData.tags}
+					digitalFiles={data.cmsAgewallData.digitalFiles}
+					roleId={data.roleId ? data.roleId : ''}
+					specifications={data.cmsAgewallData.specifications}
+					contactForms={data.cmsAgewallData.contactForms}
+					pageName={data.cmsAgewall?.title}
+					websiteLink={data.websiteLink}
+					brandName={data.brandName}
+					sessionEmail={data.email}
+					countdowns={data.cmsAgewallData.countdowns}
+					galleries={data.cmsAgewallData.galleries}
+					class={data.hideCmsZonesOnMobile ? 'hidden lg:contents' : ''}
```

Comment:
btw for the props that have the same name you should be able to do `...data.cmsAgeWallData`

```suggestion
					{...data.cmsAgeWallData}
					roleId={data.roleId ? data.roleId : ''}
					pageName={data.cmsAgewall?.title}
					websiteLink={data.websiteLink}
					brandName={data.brandName}
					sessionEmail={data.email}
					class={data.hideCmsZonesOnMobile ? 'hidden lg:contents' : ''}
```

(and brandName & websiteLink could probably be put inside a global store at layout init, like currency exchange rates, to avoid passing them around)